src/handlers/autolabel.rs

```diff
@@ -12,6 +12,27 @@ pub(super) fn parse_input(
     event: &IssuesEvent,
     config: Option<&AutolabelConfig>,
 ) -> Result<Option<AutolabelInput>, String> {
+    if event.action == IssuesAction::Opened {
```

Comment:
This is fine as an MVP, but it'd be really great if this ran after the PR was opened too. Often people change more files after a PR is opened, e.g., if CI fails.

Reply:
This matches what high-five does; I don't really have time right now to make it work for other events.

Reply:
Can't you just remove the conditional?

Reply:
> Can't you just remove the conditional?

I guess then if you tried to remove the label, rustbot would automatically read it.